doc/EnvironmentVariables.md

```diff
@@ -0,0 +1,113 @@
+| MetricUploadInterval                 | Interval at which diagnostic metrics are uploaded, defaults to **1 day**                                                 |
+| Mode                                 | specifies the mode for module deployment, **iotedged / docker / kubernetes**                                             |
+| PerformanceMetricsUpdateFrequency    | Interval to sample system performance metrics, defaults to **5 minutes**                                                 |
+| PersistentVolumeClaimDefaultSizeInMb | Size of the PersistedVolumeClaim, must be used with StorageClassName                                                     |
+| RequestTimeoutSecs                   | Timeout for handling ping and GetTaskStatus direct methods, defaults to **600**                                          |
+| RocksDB_MaxOpenFiles                 | Max number of files to be concurrently opened by RocksDB                                                                 |
+| RocksDB_MaxTotalWalSize              | Max sized to be used by RocksDB's write-ahead-log                                                                        |
+| RunAsNonRoot                         | If set, runs at user = 1000 instead of root                                                                              |
+| RuntimeLogLevel                      | Runtime diagnostic logging level, **fatal / error / warning / info / debug / verbose**, defaults to **info**             |
+| SendRuntimeQualityTelemetry          | Whether to enable sending runtime diagnostics metric, defaults to **true**                                               |
+| Storage_LogLevel                     | RocksDB diagnostic log level, **NONE / FATAL / HEADER / ERROR / WARN / INFO / DEBUG**, defaults to **NONE**              |
+| StorageClassName                     | StorageClassName to be used when creating a PersistedVolumeClaim                                                         |
+| StorageFolder                        | Path to place the EdgeHub database directory, defaults to **TempPath** of the current OS                                 |
+| UpstreamProtocol                     | Protocol used to for upstream connections, defaults to **Amqp**, and falls back to AmqpWs                                |
+| UseMountSourceForVolumeName          | ???, defaults to **false**                                                                                               |
+| UsePersistentStorage                 | Whether to save deployment config and module states to disk, defaults to **true**                                        |
+| UseServerHeartbeat                   | Sets the client-side heartbeat interval to 60sec for the Agent's upstream AMQP connection, defaults to **true**          |
+| EdgeDeviceHostName                   | omit?                                                                                                                    |
```

Comment:
All of the ones below can be omitted.

Reply:
Do we want to omit them all or just state these are reserved by edge runtime?